debian/control

```diff
@@ -117,3 +118,17 @@ Description: WebDriver driver for the Chromium Browser
  WebDriver.
  .
  See http://code.google.com/p/selenium/wiki/ChromeDriver for details.
+
+Package: chromium-browser-appmode
+Architecture: any
+Pre-Depends: dpkg (>= 1.15.6)
+Depends: ${shlibs:Depends},
+	${misc:Depends},
+	chromium-browser,
+	python3,
+	python3-gi
+Provides: mime<x-scheme-handler/webapp>
+Description: Installs a script to handle webapp://<WM_CLASS>@<URI> URIs
+ This allows us to easily run chromium in application mode from inside
```

Comment:
Should we also mention that it is needed for gvfs-open?  (Or make the comment more generic if it as more than one purpose?)